Detectors/MUON/MCH/Mapping/Interface/include/MCHMappingInterface/Segmentation.h

```diff
@@ -148,6 +148,22 @@ class Segmentation
   int mPadIndexOffset = 0;
 };
 
+/// segmentation() is a convenience function that
+/// returns a segmentation for a given detection element.
+///
+/// That segmentation is part of a pool of Segmentations
+/// (one per DE) handled by this module.
+///
+/// Note that this may be not be what you want as this module
+/// will always create one (but only one) Segmentation for
+/// all 156 detection elements.
+/// For instance, if you know you'll be dealing with only
+/// one detection element, you'd be better using
+/// the Segmentation ctor simply, and ensure by yourself
+/// that you are only creating it once in order not to incur
+/// the (high) price of the construction time of that Segmentation.
```

Comment:
Honestly, for this explanation I do not really get why using the Segmentation constructor is better than the `segmentation()` function when dealing with only one detection element...

Reply:
Indeed, I should probably sync the .h and readme for that part. Will do.